src/pooling.jl

```diff
@@ -36,8 +36,8 @@ for (front_name, backend) in (
     # We only define 3d pooling primitives, we reshape lower down to get 1d and 2d pooling
     @eval begin
         function $(Symbol("$(front_name)!"))(
-                y::AbstractArray{T,5}, x::AbstractArray{T,5},
-                pdims::PoolDims; kwargs...) where {T}
+                y::AbstractArray{T,5}, x::AbstractArray{R,5},
+                pdims::PoolDims; kwargs...) where {T, R}
```

Comment:
Just adding one comment to this file about unused type params because each function in the diff has them.